### ISSUE TITLE
Create a pause and resmue button for simulation 

### ISSUE THREAD

**Issue by JonssonGustav97** [programming-formalisms/programming_formalisms_project_autumn_2024#6]
The simulation needs a pause and a resume butto.

**Comment by richelbilderbeek** [comment]
I agree that this is part of the simulation, well done! I will set the priority to low, as it depends on having a GUI.

**Comment by richelbilderbeek** [comment]
@JonssonGustav97 as this issue has a low priority, I suggest that you unassign: I can use you better in other issues :+1:
